### ISSUE TITLE
Does not seem to support any MathJax/Latex

### ISSUE THREAD

**Issue by Hunanbean** [MSzturc/obsidian-advanced-slides#40]
It may be user error, but the slides do not seem to allow any MathJax or Latex.

**Comment by Hunanbean** [comment]
It turns out, the issue is with \ce notations. regular MathJax things work. 
for  example, this `$\ce{x Na(NH4)HPO4->[\Delta] (NaPO3)_x + x NH3 ^ + x H2O}$`
does not display correctly. It does shows the \ce, and the output, although close to what it should look like, it is not.
It instead displays as \ce${x Na(NH4)HPO4->[\Delta] (NaPO3)_x + x NH3 ^ + x H2O}$ which does not use the proper chemistry notation. It works as it should within Obsidian though. It is only when viewing as slides that things do not work properly.

**Comment by MSzturc** [comment]
Yeah im able to reproduced it. It has to do with the Library im using for rendering latex (reveal uses katex by default) i guess its time to change to mathjax.

**Comment by Hunanbean** [comment]
Thank you

**Comment by MSzturc** [comment]
will be fixed with 1.6.0
